flask_resources/serializers/simple.py

```diff
@@ -13,7 +13,7 @@
 class SimpleSerializer(BaseSerializer):
     """Simple serializer implementation."""
 
-    def __init__(self, encoder):
+    def __init__(self, encoder, **kwargs):
```

Comment:
Just wondering, not a blocker in any way:
Should we maybe store ignored arguments in a field like `self._ignored_kwargs = kwargs`?
Could that help somebody(*) with debugging, in case some keyword arguments are supplied but simply ignored?
Would that cause issues like wasted memory, or maybe even block something (idk, maybe some database models or caches or something)?

(*) most likely that would only ever benefit sysadmins who are comfortable wading through `invenio shell`...

Reply:
That's a good question. Sometimes we have arguments "leaking" that end up in kwargs 

I am not sure however how much of a performance loss or memory overhead this causes.

Reply:
it does not seem to be necessary with the latest changes. there will also be another big refactor as part of https://github.com/inveniosoftware/flask-resources/issues/117, could be re-evaluated then